.gitmodules

```diff
@@ -13,3 +13,7 @@
 	path = ml/json
 	url = https://github.com/nlohmann/json.git
 	shallow = true
+[submodule "httpd/h2o"]
+	path = httpd/h2o
+	url = https://github.com/h2o/h2o.git
+	ignore = untracked
```

Comment:
Are we sure this should not be shallow?

Reply:
we use dirty in dlib case, i just need to ignore new (untracked) files generated during build

Reply:
added shallow clone